docs/_guide/styles.md

```diff
@@ -162,327 +122,618 @@ class MyElement extends LitElement {
 }
 ```
 
-<div class="alert alert-warning">
+{:.alert .alert-info}
+<div>
 
-**Only use the `unsafeCSS` tag with trusted input.** To prevent LitElement-based components from evaluating potentially malicious code, the `css` tag only accepts literal strings. `unsafeCSS` circumvents this safeguard.
+**Only use the `unsafeCSS` tag with trusted input.** To prevent LitElement-based components from evaluating potentially malicious code, the `css` tag only accepts literal strings and numbers. `unsafeCSS` circumvents this safeguard, so use it with caution.
 
 </div>
 
-### Define styles in a style element {#style-element} 
+### Inheriting styles
 
-We recommend using static styles to style LitElement components. However, in some cases you may want to evaluate and apply styles per instance, rather than to all instances of a LitElement component. One way to do this is to include inline styles in a `<style>` element in your template, and use your element's properties in your CSS rules to evaluate styles per instance.
+Using an array of tagged template literals, a component can inherit the styles from a LitElement superclass, and add its own styles:
 
-<div class="alert alert-warning">
+```js
+class MyElement extends SuperElement {
+  static get styles() {
+    return [
+      super.styles,
+      css`...`
+    ];
+  }
+}
+```
 
-**Expressions inside a `<style>` element won't update per instance in ShadyCSS**. Due to limitations of the ShadyCSS polyfill, you can't use element properties in CSS rules as the expressions won't be evaluated.
+{% include project.html folder="style/superstyles" openFile="my-element.js" %}
 
-</div>
+## Shadow DOM styling overview {#shadow-dom}
+
+This section gives a brief overview of shadow DOM styling.
+
+Styles you add to a component can affect:
+
+* [The shadow tree (your component's rendered template)](#shadowroot).
```

Comment:
I'd flip this, as "The component's rendered template in the shadow tree."

Reply:
I think this is part and parcel of the discussion below, so not going to comment separately.